aiocache/decorators.py

```diff
@@ -310,19 +309,19 @@ async def decorator(
     ):
         missing_keys = []
         partial = {}
-        keys, new_args, args_index = self.get_cache_keys(f, args, kwargs)
+        orig_keys, cache_keys, new_args, args_index = self.get_cache_keys(f, args, kwargs)
 
         if cache_read:
-            values = await self.get_from_cache(*keys)
-            for key, value in zip(keys, values):
+            values = await self.get_from_cache(*cache_keys)
+            for orig_key, value in zip(orig_keys, values):
                 if value is None:
-                    missing_keys.append(key)
+                    missing_keys.append(orig_key)
                 else:
-                    partial[key] = value
+                    partial[orig_key] = value
             if values and None not in values:
                 return partial
         else:
-            missing_keys = list(keys)
+            missing_keys = list(orig_keys)
```

Comment:
I think you're right that the keys from `keys_from_attr` should get stored in `missing keys`, because those get passed to the callable.